src/lib/analytics/apis.web.js

```diff
@@ -33,7 +33,14 @@ const MixpanelAPI = {
 
 class GoogleWrapper {
   // eslint-disable-next-line require-await
-  async setDefaultParams(params = {}) {
+  async identify(userId) {
+    const { dataLayer } = window
+
+    dataLayer.push({ user_id: userId })
```

Comment:
does it supported through data layer or we have to use some firebase package for the web too ?